src/network/networkd-link.c

```diff
@@ -862,6 +865,8 @@ static int link_enter_set_routes(Link *link) {
 
```

Comment:
The function link_enter_set_routes() is a bit of a misnomer as soon as there's no state SETTING_ROUTES anymore

Reply:
Done